global.json

```diff
@@ -2,6 +2,6 @@
   "sdk": {
     "version": "10.0.100-rc.1.25451.107",
     "rollForward": "latestFeature",
-    "allowPrerelease": false
+    "allowPrerelease": true
```

Comment:
@lauraneto is this changed on purpose?

Reply:
Yes, in Rider it was showing that the configuration was invalid: `Possible error in Global.json: it requires a pre-release SDK version, but 'AllowPrerelease' is set to false`.
Everything still worked though, as far as I could see, but I did notice that in previous upgrades this was changed to true when using pre-release versions and then changed back.